website/source/docs/job-specification/ephemeral_disk.html.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: "docs"
+page_title: "ephemeral_disk Stanza - Job Specification"
+sidebar_current: "docs-job-specification-ephemeral_disk"
+description: |-
+  The "ephemeral_disk" stanza describes the ephemeral disk requirements of the
+  task group. Ephemeral disks can be marked as sticky and support live data
+  migrations.
+---
+
+# `ephemeral_disk` Stanza
+
+<table class="table table-bordered table-striped">
+  <tr>
+    <th width="120">Placement</th>
+    <td>
+      <code>job -> group -> **ephemeral_disk**</code>
+    </td>
+  </tr>
+</table>
+
+
+The `ephemeral_disk` stanza describes the ephemeral disk requirements of the
+task group. Ephemeral disks can be marked as sticky and support live data
+migrations. All tasks in this group will share the same ephemeral disk.
+
+```hcl
+job "docs" {
+  group "example" {
+    ephemeral_disk {
+      migrate = true
+      size    = "500"
+      sticky  = true
+    }
+  }
+}
+```
+
+## `ephemeral_disk` Parameters
+
+- `migrate` `(bool: false)` - Specifies that the Nomad client should make a
+  best-effort attempt to migrate the data from a remote machine if placement
+  cannot be made on the original node. During data migration, the task will
+  block starting until the data migration has completed.
+
+- `size` `(int: 300)` - Specifies the size of the ephemeral disk in MB.
```

Comment:
can this number be expanded later on, e.g. MySQL ephemeral_disk of 300, then run out of space, and increase to 500, will it resize the disk?